lib/hero-image-header/index.js

```diff
@@ -1,10 +1,20 @@
 import React, { Component, PropTypes } from 'react';
+var Scroll = require('react-scroll');
```

Comment:
Yes, we all _understand_ the smooth scrolling _requirement_, but right now it's not traceable.
it's *always* a good idea to leave a decision trail in the form of an issue which can be referred to in the readme... simply capturing the thought process that went into selecting *this* module should not be more than a couple of lines (*unless you want to go into detail...*)

I have a *suggestion*: 
let's merge this in now, and `then` as part of debugging https://github.com/numo-labs/isearch-ui/issues/189 (*scrolling bug*) we write a few lines of docs.